src/info.js

```diff
@@ -0,0 +1,228 @@
+/**
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+*/
+
+const path = require('path');
+const fs = require('fs-extra');
+const execa = require('execa');
+const { osInfo } = require('systeminformation');
+const { cordova, cordova_platforms: { getPlatformApi } } = require('cordova-lib');
+
+const cdvLibUtil = require('cordova-lib/src/cordova/util');
+const cdvPluginUtil = require('cordova-lib/src/cordova/plugin/util');
+
+// Cache
+let _installedPlatformsList = null;
+
+/*
+ * Sections
+ */
+
+async function getCordovaDependenciesInfo () {
+    // get self "Cordova CLI"
+    const cliPkg = require('../package');
+    const cliDependencies = await _getLibDependenciesInfo(cliPkg.dependencies);
+
+    const libPkg = require('cordova-lib/package');
+    const cliLibDep = cliDependencies.find(({ key }) => key === 'lib');
+    cliLibDep.children = await _getLibDependenciesInfo(libPkg.dependencies);
+
+    return {
+        key: 'Cordova Packages',
+        children: [{
+            key: 'cli',
+            value: cliPkg.version,
+            children: cliDependencies
+        }]
+    };
+}
+
+async function getInstalledPlatforms (projectRoot) {
+    return _getInstalledPlatforms(projectRoot).then(platforms => {
+        const key = 'Project Installed Platforms';
+        const children = Object.entries(platforms)
+            .map(([key, value]) => ({ key, value }));
+
+        return { key, children };
+    });
+}
+
+async function getInstalledPlugins (projectRoot) {
+    const key = 'Project Installed Plugins';
+    const children = cdvPluginUtil.getInstalledPlugins(projectRoot)
+        .map(plugin => ({ key: plugin.id, value: plugin.version }));
+
+    return { key, children };
+}
+
+async function getEnvironmentInfo () {
+    const [npmVersion, osInfoResult] = await Promise.all([_getNpmVersion(), osInfo()]);
+    const { platform, distro, release, codename, kernel, arch, build } = osInfoResult;
+
+    const optionalBuildSuffix = build ? ` (${build})` : '';
+
+    const osFormat = [
+        platform === 'darwin' ? codename : distro,
+        release + optionalBuildSuffix,
+        build ? `(${build})` : '',
```

Comment:
```suggestion
```

`build` is already contained in `optionalBuildSuffix`. I probably messed up during refactoring.